deploy/playbooks/inventories/group_vars/kind.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Is this file used?

Reply:
yes, it's a place if we need to add local group variables for kind deployment